### ISSUE TITLE
flood of `set-output` deprecation warnings

### ISSUE THREAD

**Issue by patriziobrunops** [GitTools/actions#758]
> Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

My workflow summaries are getting flooded by the above message, which is muddling the workflow summary making it difficult to read. I'm using `gittools actions` version `0.9.14`, which seem to be using `@actions/core v1.10.0` as indicated by the blog post :thinking:

**Comment by arturcic** [comment]
@patriziobrunops is your workflow public? overwise can you post the part of the workflow starting with checkout up to the gittools actions?

**Comment by patriziobrunops** [comment]
```yaml
name: Set Version

on:
  workflow_call:
    inputs:
      targetPath:
        type: string
        description: Optionally supply the path to the working directory.
        default: .
        required: false
      configFilePath:
        type: string
        description: Optional path to config file (defaults to GitVersion.yml).
        default: GitVersion.yml
        required: false
    outputs:
      sha: 
        value: ${{ jobs.setup-version.outputs.Sha }}
      shortSha:
        value: ${{ jobs.setup-version.outputs.ShortSha }}
      semver: 
        value: ${{ jobs.setup-version.outputs.semVer }}

jobs:
  setup-version:
    runs-on: ubuntu-latest
    outputs:
      sha:  ${{ steps.gitversion.outputs.Sha }}
      shortSha: ${{ steps.gitversion.outputs.ShortSha }}
      semVer: ${{ steps.gitversion.outputs.semVer }}
    steps:
      - uses: actions/checkout@v3
        with:
          fetch-depth: 0
      - name: Install GitVersion
        uses: gittools/actions/gitversion/setup@v0.9.14
        with:
          versionSpec: '5.x'
      - name: Determine Version
        id: gitversion
        uses: gittools/actions/gitversion/execute@v0.9.14
        with:
          useConfigFile: true
          targetPath: ${{ inputs.targetPath }}
          configFilePath: ${{ inputs.configFilePath }}
```

**Comment by vktr** [comment]
I also have the same behavior in my actions.

Build output: https://github.com/porla/core/actions/runs/3347499683/jobs/5545550773 (check the _Install GitVersion_ and _Calculate version_ steps).

Workflow: https://github.com/porla/core/blob/main/.github/workflows/ci.yaml

**Comment by arturcic** [comment]
@patriziobrunops, @vktr fixed in version 0.9.15, ca you confirm?

**Comment by vktr** [comment]
@arturcic Confirmed - thanks!
